apps/server/src/api/v2/file-upload.ts

```diff
@@ -14,3 +21,17 @@ function handleFileUpload(ctx) {
     },
   };
 }
+
+function delPrevFilesIfAny(currentFileName) {
+  const uploadsDir = config.uploadsPath;
+  return getFileNames(uploadsDir).then(files => {
```

Comment:
can we please try to avoid mixing `promise.then` syntax with async/await in the same function? just for the sake of consistency, after all they're the same thing. I prefer async/await since it usually looks cleaner as it avoid nesting, ex:

```typescript
const files = await getFileNames(uploadDir);
const resourceId = currentFileName.split('-')[0];
const filesToDel = files.filter(...);
```